iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/model/ConfigMap.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.kubernetes.model;
+
+import java.util.Map;
+
+import org.sonar.iac.common.yaml.tree.YamlTree;
+
+public record ConfigMap(
+  String path,
```

Comment:
The `filePath` will be a better name like in `ServiceAccount`.
I guess that the `name` property is missing here.
The same in `MapResource` and `Secret`.